docs/UserGuide.md

```diff
@@ -61,35 +61,31 @@ Teaching Assistant Manager (TAManager) is a desktop application for managing tea
 </div>
 
 <span style="float:right; font-size: 0.8em;">[BACK TO TOP](#table-of-contents)</span>
-### Viewing help : `help`
-
-Shows a message explaining how to access the help page.
-
-Format: `help`
 
-![help message](images/helpMessage.png)
+## Teaching Assistant Commands
 
 ### Adding a Teaching Assistant: `add`
 
 Adds a new teaching assistant to TAManager.
 
-Format: `add n/NAME p/PHONE e/EMAIL tele/TELEGRAM [t/TAG]... [c/COURSE_CODE]... h/HOUR`
+Format: `add n/NAME p/PHONE e/EMAIL tele/TELEGRAM h/HOUR [t/TAG]... [c/COURSE_CODE]...`
 
 - `NAME` should be a string.
 - `PHONE` should be a 3-10 digits integer.
 - `EMAIL` should contain "@".
 - `TELEGRAM` should be between 5-32 characters and start with "@", and it cannot contain any special characters except underscore.
+- `HOUR` should be an integer.
 - `TAG` should be an alphanumeric string without spaces.
 - `COURSE_CODE` should start with 2-3 alphabets, followed by 4 numbers, and optionally end with an alphabet.
-- `HOUR` should be an integer.
 
 Example:
-`add n/ Rayner Toh p/93812311 e/rayner@example.com tele/@raynertjx t/parttime c/CS2103T h/4` will add a new teaching assistant named Rayner Toh to TAManager.
+`add n/ Rayner Toh p/93812311 e/rayner@example.com tele/@raynertjx h/4 t/parttime c/CS2103T` will add a new teaching assistant named Rayner Toh to TAManager.
 
 ![add TA](images/addTA.png)
```

Comment:
Would be good to update this screenshot

Reply:
is this screenshot outdated?

Reply:
I think so, new TAs don't have free timeslots